histomicsui/web_client/panels/DrawWidget.js

```diff
@@ -170,6 +192,73 @@ var DrawWidget = Panel.extend({
         });
     },
 
+    /**
+     * Respond to a click on the "view" button by changing the
+     * viewer location and zoom level to focus on one annotation
+     */
+    viewElement(evt) {
+        const annot = this.collection._byId[$(evt.target).parent().attr('data-id')];
+        let points;
+        let pointAnnot = false;
+        switch (annot.get('type')) {
+            case 'point':
+                points = [annot.get('center')];
+                pointAnnot = true;
+                break;
+            case 'polyline':
+                points = annot.get('points');
+                break;
+            case 'rectangle':
+                points = convertRectangle(annot.attributes).coordinates[0];
+                break;
+            case 'ellipse':
+                points = convertEllipse(annot.attributes).coordinates[0];
+                break;
+            case 'circle':
```

Comment:
We have more annotation types that should ultimately be handled (image overlays, pixelmaps, heatmaps, grid data) -- that can be a future PR (or don't show the control on those elements for now).

Reply:
The button won't be show on other annotation types for now.

See 863978e